cypress/integration/v2/features/dynamic-content/site-data.spec.js

```diff
@@ -0,0 +1,135 @@
+
+/**
+ * External dependencies
+ */
+import { registerTests } from '~stackable-e2e/helpers'
+
+describe( 'Dynamic Content - Site', registerTests( [
+	matchSiteData,
+	adjustFieldOptions,
+	adjustFieldValues,
+] ) )
+
+const fields = [
+	{
+		name: 'Site Title',
+		defaultValue: 'e2etest',
+		testValue: 'Test Site Title',
+	},
+	{
+		name: 'Site Tagline',
+		defaultValue: 'Just another WordPress site',
+		testValue: 'Test Site Tagline',
+	},
+	{
+		name: 'Site URL',
+		defaultValue: Cypress.config( 'baseURL' ),
+		testValue: 'Test Site Url',
+	},
+]
+
+const adjustSiteField = ( fieldName, fieldOptions = {} ) => {
+	 cy.adjustDynamicContent( 'ugb/cta', 0, '.ugb-cta__title', {
+		 source: 'Site',
+		 fieldName,
+		 fieldOptions,
+	 } )
+}
+
+function matchSiteData() {
+	 it( 'should match dynamic content in site fields', () => {
+		 cy.setupWP()
+
+		 fields.forEach( fields => {
+			 cy.newPost()
+			 cy.addBlock( 'ugb/cta' )
+
+			const fieldOptions = {}
+			adjustSiteField( fields.name, fieldOptions )
+
+			// Asserting content value with assertBlockContent
+			cy.openInspector( 'ugb/cta', 'Style' )
```

Comment:
Pull master to this branch & remove `cy.openInspector( 'ugb/cta', 'Style' )`
in this spec file, we don't need this anymore